src/Loggers/LogGraph.jl

```diff
@@ -2,7 +2,11 @@
 	log_graph
 """
 function log_graph(logger::TBLogger, name::AbstractString, g::AbstractGraph; step = nothing, nodelabel::Vector{String} = map(string, vertices(g)), nodeop::Vector{String} = map(string, vertices(g)), nodedevice::Vector{String} = fill("cpu", nv(g)), nodevalue::Vector{Any} = fill(nothing, nv(g)))
-	summ = SummaryCollection(graph_summary(name, g, nodelabel, nodeop, nodedevice , nodevalue))
+	@assert nv(g) == length(nodelabel) "length of nodelable must be same as number of vertices"
+	@assert nv(g) == length(nodeop) "length of nodeop must be same as number of vertices"
+	@assert nv(g) == length(nodedevice) "length of nodedevice must be same as number of vertices"
+	@assert nv(g) == length(nodevalue) "length of nodevalue must be same as number of vertices"
```

Comment:
There should be `nv(g) == length(foo) || throw(ArgumentError(...`

Assertions are not for input checking.